bolt-modules/boltlib/types/argsspec.pp

```diff
@@ -0,0 +1,10 @@
+# SensitiveData represent the acceptable types to pass as arguments into the run_xxx()
```

Comment:
Looks like the name may have changed to ArgsSpec. Can you update the comment?

Reply:
Fixed, thanks!